pkg/network/tracer/tracer.go

```diff
@@ -341,6 +342,11 @@ func (t *Tracer) Stop() {
 	t.ebpfTracer.Stop()
 	t.httpMonitor.Stop()
 	t.conntracker.Close()
+
+	// Reset the global telemetry registry
+	// This has to be done to avoid panic in
```

Comment:
Does this reset everything in the package? Would this cause problems for other tests?

Reply:
I should reword the comment.

It resets the global registry maintained by the prometheus packages.